networks/deescord.py

```diff
@@ -130,32 +133,49 @@ class Discord(object):
 
 
     async def retweet(self, message):
+        screen_name = message.content.split('|retweet')[1].strip()
 
-        tooter = message.content.split('|retweet')[1]
-        if not tooter:
+        if not screen_name:
             return await self.say(message.channel, 'Who should I retweet?')
 
-        tooter = tooter.strip()
+        else:
```

Comment:
not a big deal, but an anti-pattern to keep an eye out for:
```
if error_condition:
    return
else:
    stuff
```

v. 

```
if error_condition:
    return

stuff
```

it's nice to strip out that `else` and the extra level of indentation in the rest of the method